Thumbnail source can use an icon (#3328)
Shopify/polaris-react: src/components/Thumbnail/README.md

/>
```

### Thumbnail with component source

> Use when a thumbnail uses a component inside of it.

Reviewer: ```suggestion
Use to render an icon inside of thumbnail.
```
